custom_components/zwave_mqtt/light.py

```diff
@@ -61,6 +60,8 @@ def value_added(self):
     @property
     def brightness(self):
         """Return the brightness of this light between 0..255."""
+        if self.values.get("Target") is not None:
```

Comment:
Can we test `self.values.target is not None` ?

Reply:
not sure, isn't that a nullptr if it doesn't exist ? It's an optional attribute, not present on many lights.